src/registrar/config/urls.py

```diff
@@ -11,9 +11,12 @@
 from registrar.views import health, index
 
 urlpatterns = [
+    path("", index.index, name="home"),
     path("admin/", admin.site.urls),
-    path("", index.index),
     path("health/", health.health),
+    path("openid/", include("djangooidc.urls")),
+    # these views respect the DEBUG setting
+    path("__debug__/", include("debug_toolbar.urls")),
 ]
 
 if settings.DEBUG:
```

Comment:
**suggestion**: I think that we can get rid of the entire `if` block here because adding the url patterns above does the same thing in all cases.

Reply:
Good catch. This was leftover mess from a less-than-careful rebase.